apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -167,8 +166,8 @@ function EventTypeSingleLayout({
 
   const { isManagedEventType, isChildrenManagedEventType } = useLockedFieldsManager(
     eventType,
-    t("locked_fields_admin_description"),
-    t("locked_fields_member_description")
+    formMethods,
```

Comment:
We now need form methods in order to set discretely the different fields in the event type metadata